sdk/README.md

```diff
@@ -0,0 +1,39 @@
+# Lookout Analyzer SDK
+
+An analyzer should implement [gRPC service](https://github.com/src-d/lookout/tree/master/pb/service_analyzer.proto) to receive events from the server.
```

Comment:
For people who might not be very proficent with gRPC, how about making server/client separation more explicit  and then making both links to .proto to have a consistent wording, like

`An analyzer is a gRPC server, that implements [Analyzer service]() ...`

and then

`An analyzer should use gRPC client to access [Data service]() to ...`